src/frontend/components/check-list/check-list.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { ReactNode, useState } from 'react';
+import PageBlock from '../../layouts/page-block/page-block';
+import { Form, Button, Modal } from 'react-bootstrap';
+import styles from './check-list.module.css';
+
+interface CheckListProps {
+  title: string;
+  headerRight?: ReactNode;
```

Comment:
What is this prop for? I'm not seeing it used in the risk log file?

Reply:
I tried to model this component to be modeled after the BulletList component, which also had it, so I figured I would add it in here too (I'm assuming you mean the headerRight prop?)

Reply:
yeah lets keep it for the same reason bulletlist has it, just in case